modules/core/client/views/subjectinfo.client.view.html

```diff
@@ -48,12 +48,11 @@ <h1 class="text-center">{{info.fullName}}</h1>
               
           <!-- Display the contact info -->
           <p>Name: {{info.emergencyContact1.name}}</p>
-          <p>Name: {{info.emergencyContact2.name}}</p>
-
           <p> Relationship: {{info.emergencyContact1.relationship}}</p>
-          <p> Relationship: {{info.emergencyContact2.relationship}}</p>
-
           <p> Phone: {{info.emergencyContact1.phone}}</p>
+          </br>
```

Comment:
!fix Use margin css to fix vertical spacing, not br. Usually, br is done within text and paragraphs to move to the next line, not to put space between paragraphs themselves